### PR TITLE
Add link to SSG for admonitions

### PR DESCRIPTION
**Pull request by mburke5678**
Add a link to the Supplementary Style Guide for definitions of Admonitions.

Preview: [Admonitions](https://github.com/openshift/openshift-docs/blob/e445b2cd207c396eeb35dd1b0a992d8a0adadb1d/contributing_to_docs/doc_guidelines.adoc#admonitions)

### CODE REVIEW

File: contributing_to_docs/doc_guidelines.adoc [hunk 1]
```diff
@@ -1650,6 +1650,8 @@ Text for admonition
 ====
 ....
 
+See the link:https://redhat-documentation.github.io/supplementary-style-guide/#admonitions[Red Hat Supplementary style guide for the valid admonition types and their defiitions].
```
Comment (skopacz1) [inline contributing_to_docs/doc_guidelines.adoc:1653]: Suggestion:
```suggestion
See the link:https://redhat-documentation.github.io/supplementary-style-guide/#admonitions[Red Hat Supplementary style guide] for the valid admonition types and their definitions.
```
Up to you! I just feel that limiting the link text to the name of the destination is specific enough without making it too long.

Comment (skopacz1) [inline contributing_to_docs/doc_guidelines.adoc:1653]: typo: defiitions -> definitions

**Review by skopacz1 — COMMENTED** [review]
A few nits but otherwise looks good! 

I'm still new to the review squad, so please wait until @adellape gives my review the go-ahead before implementing any feedback. Thanks!

**Comment by adellape** [comment]
@skopacz1 @adellape Peer review looks good 👍

**Pull request merged**